test/spec/modules/essensBidAdapter_spec.js

```diff
@@ -0,0 +1,796 @@
+import { expect } from 'chai';
+import adapter from 'modules/essensBidAdapter';
+import bidmanager from 'src/bidmanager';
+import adLoader from 'src/adloader';
+describe('Essens adapter tests', function () {
+  describe('Test callbid method ', function () {
+    let stubLoadScript
+    beforeEach(() => {
+      stubLoadScript = sinon.stub(adLoader, 'loadScript');
+    });
+
+    afterEach(() =>
+      stubLoadScript.restore()
+    );
+
+    it('invalid case: missing bid', () => {
```

Comment:
I can't think of any case where bids would be missing.  But there should be cases where bad bids are sent, i.e. bad bid.params.  So I'm not sure this is a good test.